tools/run_tests_docker.sh

```diff
@@ -138,31 +138,22 @@ if test ${init_only}; then
     exit
 fi
 
-if test ${special}; then
-    echo 'Using the special config and only running test_dirac'
-    tools/pytest.sh -v --tb=short ${exitfirst:-} test_dirac.py
+PYTEST_SH_ARGS="${exitfirst:-}"
+
+if test ${trace}; then
+    echo 'Running tests in verbose mode'
+    PYTEST_SH_ARGS="-vvv $PYTEST_SH_ARGS"
 else
-    if test ${trace}; then
-        echo 'Running tests in verbose mode'
-        if test ${votest}; then
-            echo "Running the following votests for ${POLICY}"
-            TESTS=$(echo "${@:2}")
-            echo $TESTS | tr " " "\n"
-            tools/pytest.sh -vvv ${exitfirst:-} $TESTS
-        else
-            tools/pytest.sh -vvv ${exitfirst:-}
-        fi
-    else
-        echo 'Running tests'
-        if test ${votest}; then
-            echo "Running the following votests for ${POLICY}"
-            TESTS=$(echo "${@:2}")
-            echo $TESTS | tr " " "\n"
-            tools/pytest.sh -v --tb=short ${exitfirst:-} $TESTS
-        else
-            tools/pytest.sh -v --tb=short ${exitfirst:-}
-        fi
-    fi
+    echo 'Running tests'
+    PYTEST_SH_ARGS="-v --tb=short $PYTEST_SH_ARGS"
+fi
+
+if test ${selection}; then
+    echo "Running the following tests:"
+    echo $TESTS | tr " " "\n"
+    tools/pytest.sh $PYTEST_SH_ARGS $TESTS
```

Comment:
Quick question @bziemons: The TESTS env var should contain 1) the full path to the test or 2) path relative to the tests dir?

Reply:
relative path, afaik (see the example in the PR post). It is how pytest interprets it and pytest is run on the test directory.